test/test_html.rb

```diff
@@ -39,8 +39,14 @@ def test_works_with_multiple_children
   end
 
   def test_works_with_void_tag
-    subject = ::Scarpe::HTML.render { |h| h.input(type: "text") }
+    subject = Scarpe::HTML.render { |h| h.input(type: "text") }
 
     assert_equal '<input type="text" />', subject
   end
+
+  def test_raises_with_void_tag_blocks
```

Comment:
@TobiasBales gotcha a little test there! 

Also I don't think we need the `::Scarpe` so touched thhat